src/detectors/BHCAL/BHCAL.cc

```diff
@@ -59,7 +59,7 @@ extern "C" {
             .sampFrac = 0.033, // average, from sPHENIX simulations
             .readout = "HcalBarrelHits",
             .layerField = "tower",
```

Comment:
```suggestion
            .layerField = "",
```

Reply:
Ah! Good catch: I forgot that putting this will sum all the tiles in a tower...